src/ontology/mondo-edit.obo

```diff
@@ -110738,7 +110738,6 @@ intersection_of: disease_has_location UBERON:0001707 ! nasal cavity
 [Term]
 id: MONDO:0006315
 name: neoplastic medium-sized B-lymphocyte with basophilic cytoplasm
```

Comment:
Did you intend to simply delete the definition of this term here rather than replacing it?

Reply:
yeah, there is no def in NCIT and I think this term should be obsoleted anyway (see #2099)